buddyboss/buddyboss-platform
# Forums - when viewing another user's forum replies, it says "My Replies"

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #882 .

### How to test the changes in this Pull Request:

1. Logged into the site.
2. View logged in user's profile. 
3. Go to Forums Tab see the sub navigations.
4.  Try to view the same thing for other members.

### Proof Screenshots or Video
Logged in user: http://prntscr.com/sfd3p7
Member: http://prntscr.com/sfd36c



### Other information:
